stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -445,6 +447,23 @@ public class ChatClient internal constructor(
         return api.searchMessages(request)
     }
 
+    public fun getFileAttachments(channelType: String, channelId: String, offset: Int, limit: Int): Call<List<Attachment>> =
```

Comment:
Nice one! Should we also add those methods to `ChannelClient`?

Reply:
Good point, let me add it